executors.go

```diff
@@ -70,6 +122,29 @@ func (c *Connection) Save(model interface{}, excludeColumns ...string) error {
 // ValidateAndCreate applies validation rules on the given entry, then creates it
 // if the validation succeed, excluding the given columns.
 func (c *Connection) ValidateAndCreate(model interface{}, excludeColumns ...string) (*validate.Errors, error) {
+	v := reflect.Indirect(reflect.ValueOf(model))
```

Comment:
I feel like a lot of this logic is repeated. would make sense to break it out into it's own method?